libpod/util_linux.go

```diff
@@ -91,3 +92,19 @@ func GetV1CGroups(excludes []string) cgroups.Hierarchy {
 		return filtered, nil
 	}
 }
+
+// LabelVolumePath takes a mount path for a volume and gives it an
+// selinux label of either shared or not
+func LabelVolumePath(path string, shared bool) error {
```

Comment:
Why are you passing in shared, and never using it?

Reply:
ah was supposed to be using it, fixed